bigchaindb/lib.py

```diff
@@ -462,7 +462,10 @@ def fastquery(self):
 
     def get_validators(self):
         try:
-            resp = requests.get('{}validators'.format(self.endpoint))
+            block = self.get_latest_block()
```

Comment:
Worth adding a test case in `tests/tendermint/test_integration.py`. Something like:

```
query.store_validator_update -> query.store_validator_update -> assert -> query.delete_validator_update  -> assert
```

What do you think?